### ISSUE TITLE
torch_geometric.nn.norm.batch_norm OR torch.nn.BatchNorm1d?

### ISSUE THREAD

**Issue by cowry5** [rusty1s/pytorch_geometric#809]
## ❓ Questions & Help

Are these two methods equivalent if I want to apply batch normalization to the output of the convolution?

**Comment by rusty1s** [comment]
Yes :)
